_drgn.pyi

```diff
@@ -453,6 +453,32 @@ class Program:
             return an :class:`Object` or ``None`` if not found.
         """
         ...
+    def add_symbol_finder(
+        self, fn: Callable[[Optional[str], Optional[int], bool], Sequence[Symbol]]
+    ) -> None:
+        """
+        Register a callback for finding symbols in the program.
+
+        The callback should take three arguments: a search name, a search
+        address, and a boolean flag 'first' indicating whether to return only
+        the first match. When the 'first' flag is True, the callback should
+        return just one :class:`Symbol`. When the flag is False, the callback
+        should return a list of all matching :class:`Symbol`\\ s. Both the name
+        and address arguments are optional. If both are provided, then the
+        result(s) should match both. If neither are provided, the finder should
+        return all available symbols.
+
+        Callbacks are called in reverse order of the order they were added
+        (i.e,, the most recently added callback is called first). When the
+        'first' flag is set, the search will short-circuit after the first
+        finder which returns a result, and subsequent finders will not be
+        called. Otherwise, all callbacks will be called, and all results will be
+        returned.
+
+        :param fn: Callable taking name, address, and 'first' flag, and
+            returning a sequence of :class:`Symbol`\\ s.
```

Comment:
Could you rephrase this to call the `first` argument `one` instead, both to match the libdrgn documentation and indicate that it's not necessarily the first match, but the "best" match?